app/lib/frontend/handlers/account.dart

```diff
@@ -14,6 +17,62 @@ import '../../shared/handlers.dart';
 
 import '../templates/admin.dart';
 
+/// Handles POST /api/account/session
+Future<shelf.Response> updateSessionHandler(shelf.Request request,
+    {ClientSessionData clientSessionData}) async {
+  final user = await requireAuthenticatedUser();
+
```

Comment:
Let's put an `access_token` in the request body, validate it and do a lookup the userinfo service.